fulltext/backends/__ocr.py

```diff
@@ -0,0 +1,52 @@
+# sudo apt-get install tesseract-ocr
+# sudo pip3 install pytesseract
+# sudo apt-get install tesseract-ocr-[lang]
+
+from fulltext.util import which
+import pytesseract
+from PIL import Image
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+EXTENSIONS = ('jpg', 'jpeg', 'bmp', 'png', 'gif')
+
+if which('tesseract') is None:
+    LOGGER.warning('CLI tool "tesseract" is required for image files backend.')
+
+
+def read(img, **kargs):
+    lang = kargs.get('lang', 'eng')
+
+    im = Image.open(img)
+
+    degree = 0
+    try:
+        exif = im._getexif()
+    except AttributeError:
+        exif = None
+
+    if exif:
+        orientation_key = 274  # cf ExifTags
+        if orientation_key in exif:
+            orientation = exif[orientation_key]
+            rotate_values = {
+                3: 180,
+                6: 270,
+                8: 90
+            }
```

Comment:
`rotate_values` also seems like it should be a constant. You could optionally condense all of this code to:

```python
try:
    exif = im._getexif()
except:
    # No EXIF data, no rotation necessary.
    pass
else:
    degree = EXIF_ROTATION.get(exif.get(EXIF_ORIENTATION, None), None)
    if degree:
        im = im.rotate(degree)
```